src/main/UseCases/AddOutputBoundary.java

```diff
@@ -0,0 +1,12 @@
+package UseCases;
+
+public interface AddOutputBoundary {
```

Comment:
The AddPresenter can implement interface, and I think this OutputBoundary records all the changes the AddUseCase class does, and then presenter knows what to present.